packages/elements/src/tooltip/index.ts

```diff
@@ -252,9 +256,11 @@ class Tooltip extends BasicElement {
 
   /**
    * Clear all timers
+   * @param enter Indicates whether the tooltip should be reset due to a mouse leave event
    * @returns {void}
    */
-  private reset = (): void => {
+  private reset = (enter?: boolean): void => {
```

Comment:
use default value instead
```suggestion
  private reset = (enter = false): void => {
```